Form/Type/ChangePasswordFormType.php

```diff
@@ -36,7 +36,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'form.current_password',
             'translation_domain' => 'FOSUserBundle',
             'mapped' => false,
-            'constraints' => new UserPassword(),
+            'constraints' => new UserPassword(array('groups' => $options['validation_groups'])),
```

Comment:
this is broken in case multiple groups are passed in the option, as the constraint will now run multiple times (once per group). You should take only the first validation group from the options